include/Reader/readerir.h

```diff
@@ -1317,6 +1325,7 @@ class GenIR : public ReaderBase {
   uint32_t TargetPointerSizeInBits;
   const uint32_t UnmanagedAddressSpace = 0;
   const uint32_t ManagedAddressSpace = 1;
+  static uint32_t TailCallCount;
 };
```

Comment:
In this I just modeled the current behavior from Jit64.  My hand wavier was that we'd run in "single threaded mode".  That being said I don't know how to tell CoreCLR to do that.  But I agree we should figure this out.